posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+
+![Open With](/images/spyder-files-open-with.png)
+
+## Single click open on file explorer
+
+Another improvement made was the option to open files and directories with a single click which was not possible before because files and folders could only be opened by double clicking them. In order to enable this option, go to the files tab in the preferences window, check the box “Single click to open files” and apply the changes. With this set, now only a single click is needed to open a file externally or in Spyder, from the files tab. 
+
+![Single click](/images/spyder-files-single-click.png)
+
+Bear in mind that changing this configuration option will also affect the behaviour of the Project Explorer pane.
+
+## Open files externally
+
+Additionally, we added a context menu action called Open externally to open files with the Operating System default program associated with the file type. To use this action, go to the file you want to open in the files tab and right-click it to see the context menu. Then select “Open Externally” and the file will be opened outside Spyder, with the default program associated with its file type, depending on the Operating System.
+
+![Open Externally](/images/spyder-files-open-externally.png)
+
+Moreover, now it is possible to select several files to perform a specific action thanks to the multi-select functionality added. For this, press CTRL or SHIFT and Click on the files that are going to be selected without releasing the CTRL/SHIFT. It is also possible to select the files without doing it one by one, but selecting a file with the CTRL/SHIFT clicked and selecting a not contiguous file to this one which will select all the files between them.
+By doing this, it is possible to execute some of the actions available in the context menu for all the files selected including delete, rename, move, copy and open externally.
+
+
+![Select Files](/images/spyder-files-select-files.png)
+
+## Absolute and relative path handling
+
+Another functionality added was the ability to copy and paste files and their absolute or relative paths. To access these actions, go to the context menu by right-clicking a file in the files tab. 
+
+![Copy Path](/images/spyder-files-copy-path.png)
+
+With this, a file can be copied from the files tab and pasted anywhere else directly from Spyder. The “Copy Absolute Path” and “Copy Relative Path” actions, give us access to the files’ path, to be pasted as text. 
+
+This way, when pasting the absolute path of a file, we get the complete path of the file starting from the root.
+
+![Absolute Path](/images/spyder-files-absolute-path.png)
+
+When using the option “Copy Relative Path”, we get the path of the file relative to the directory in which we are located.
```

Comment:
```suggestion
Alternatively, `Copy Relative Path` gets us the path of the file relative to the current working (displayed in the top right of the Spyder window).
```

* Fix grammar
* Clarify explanation
* Make description more accurate
* Use proper UI text and terminology
* Use verbatim instead of quotes